modules/exploits/multi/http/phpmailer_arg_injection.rb

```diff
@@ -60,7 +60,10 @@ def initialize(info = {})
       ])
     register_advanced_options(
       [
-        OptInt.new('WAIT_TIMEOUT', [true, 'Seconds to wait to trigger the payload', 300])
+        OptInt.new('WaitTimeout', [true, 'Seconds to wait to trigger the payload', 300]),      
+        OptString.new('NameField', [true, 'Name of the element for the Name field', 'name'], regex: /^([^\t\n\f \/>"'=]+)$/),
+        OptString.new('EmailField', [true, 'Name of the element for the Email field', 'email'] regex: /^([^\t\n\f \/>"'=]+)$/),
+        OptString.new('MessageField', [true, 'Name of the element for the Message field', 'message'] regex: /^([^\t\n\f \/>"'=]+)$/)
```

Comment:
Looks like a syntax error here is causing the unit tests to fail.

```suggestion
        OptString.new('EmailField', [true, 'Name of the element for the Email field', 'email'], regex: /^([^\t\n\f \/>"'=]+)$/),
        OptString.new('MessageField', [true, 'Name of the element for the Message field', 'message'], regex: /^([^\t\n\f \/>"'=]+)$/)
```

Reply:
Thanks!